### PR TITLE
Add rule to manage tech victory difficulty

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

**Comment by geoffthemedio** [comment]
Looks OK

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -1190,6 +1190,12 @@ Tech Cost Scaling
 RULE_TECH_COST_FACTOR_DESC
 Adjusts costs of tech
 
+RULE_SINGULARITY_COST_FACTOR
+Singularity of Transcendence tech Cost Scaling
```
Comment (Cjkjvfnby) [inline default/stringtables/en.txt]:
```suggestion
[[LRN_TRANSCEND]] Cost Scaling
```

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -1190,6 +1190,12 @@ Tech Cost Scaling
...
+
+RULE_SINGULARITY_COST_FACTOR_DESC
+Adjusts costs of Singularity of Transcendence tech
```
Comment (Cjkjvfnby) [inline default/stringtables/en.txt]:
```suggestion
 Adjusts costs of [[LRN_TRANSCEND]] tech
```

File: default/stringtables/en.txt (hunk 1 above)
Comment (o01eg) [reply default/stringtables/en.txt]: Fixed

File: default/stringtables/en.txt (hunk 2 above)
Comment (o01eg) [reply default/stringtables/en.txt]: Fixed

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**